### PR TITLE
feat(Badge, Tag): add badge to Tag and fix Badge's border and fontSize scaling

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issues: [WEB-1733](https://jira.tid.es/browse/WEB-1733), [WEB-1734](https://jira.tid.es/browse/WEB-1734) and [WEB-1738](https://jira.tid.es/browse/WEB-1738)

### CODE REVIEW

**Review by AnaMontes11 — APPROVED** [review]
LGTM!

File: src/__stories__/tag-story.tsx [hunk 1]
```diff
@@ -12,19 +12,29 @@ import {
     Box,
 } from '..';
 
+const badgeOptions = ['0', '2', '14', 'true', 'false'];
```
Comment (yceballost) [inline src/__stories__/tag-story.tsx:15]: Inconsistency with Avatar story
https://mistica-web.vercel.app/?path=/story/components-avatar--default

image

**Review by yceballost — APPROVED** [review]

File: src/__stories__/tag-story.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/__stories__/tag-story.tsx]: fixed

**Review by pladaria — APPROVED** [review]

**Pull request merged**